ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -322,6 +465,29 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-confidential-trustedlaunch
+- as: azure-ipi-des-day2-arm-f28
+  cron: 17 22 1 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      OCP_ARCH: arm64
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-azure-ipi-disk-encryption-set-day2
+- as: azure-ipi-des-mini-perm-arm-f28
```

Comment:
Similar comment with https://github.com/openshift/release/pull/54740#discussion_r1698212836

Reply:
ok, let's remove azure-ipi-des-mini-perm-amd-mixarch-f28

Reply:
Seem like this one is duplicated with `azure-ipi-des-mini-perm-fips-amd`